package.json

```diff
@@ -13,13 +13,16 @@
   },
   "scripts": {
     "prepare": "npm run build",
-    "start": "npm run dev",
+    "set-version": "node ./set-version-env.js",
+    "start": "export $(npm run -s set-version) && npm run dev",
     "dev": "npm run build:debug && miniflare dist/worker.mjs --watch --debug -m --r2-persist --global-async-io --global-timers",
-    "build": "esbuild --bundle src/index.js --format=esm --sourcemap --minify --outfile=dist/worker.mjs && npm run build:tsc",
-    "build:debug": "esbuild --bundle src/index.js --format=esm --outfile=dist/worker.mjs",
+    "build": "export $(npm run -s set-version) && esbuild --bundle src/index.js --format=esm --sourcemap --minify --outfile=dist/worker.mjs && npm run build:tsc",
```

Comment:
question: should these just be added to `prepare` above? it would be nice to avoid duplicating this line, and seems like `set-version` can probably safely be run any time `prepare` is triggered - does that sound right to you?

Reply:
Yes, that sounds right. I will update that. thanks

Reply:
ah hold on, prepare task is run when you install the packages, but we can improve this code for sure.